### DISCUSSION TITLE
How to write classgraph license in other project?

### DISCUSSION THREAD

**Discussion by worldbiomusic** [classgraph/classgraph#622] [Q&A] [answered]
Hi, I'm using `classgraph` jar file in my [project](https://github.com/MiniGameWorlds/MiniGameWorld)([detailed source code](https://github.com/MiniGameWorlds/MiniGameWorld/blob/main/src/com/worldbiomusic/minigameworld/listeners/CommonEventListener.java)) which use also MIT license

But do I have to reference classgraph github in the document?

Or just need the same license as MIT?

**Comment by lukehutch** [comment, marked as answer]
The MIT license is compatible with basically every other license in existence, although it may not be true in the opposite direction, for example if you are library is licensed as GPL without the classpath exception.

You don't need to reference ClassGraph anywhere, you can just include it as a dependency. If you want to reference ClassGraph on your own project page, you can just link to the ClassGraph GitHub home page.

**Reply by worldbiomusic** [reply]
In next commit, I'll write classgraph with license in the wiki docs.

Thanks.
